paimon-core/src/test/java/org/apache/paimon/mergetree/MergeTreeTest.java

```diff
@@ -195,47 +197,54 @@ public static void after() {
         service = null;
     }
 
-    @Test
-    public void testEmpty() throws Exception {
-        doTestWriteRead(0);
+    @ParameterizedTest
+    @EnumSource(SortEngine.class)
```

Comment:
If you need `parameterized` annotation for all tests, I would suggest making this an abstract test class and create two subclasses. One subclass for loser tree and another for min heap.